spec/features/users/user_signs_up_spec.rb

```diff
@@ -15,6 +15,7 @@
   scenario 'provide valid registration information' do
     visit new_user_registration_path
 
+    fill_in 'Name', with: "John P. Example"
```

Comment:
If this is a username instead of name name, it might make sense to have an example that seems more like a username than a name.